### ISSUE TITLE
#createRefactoring:for: should be moved up to Slot

### ISSUE THREAD

**Issue by MarcusDenker** [pharo-ide/Calypso#482]
see https://github.com/pharo-project/pharo/issues/3072

**Comment by guillep** [comment]
Why is this a calypso thing? It should be part of the refactoring browser

**Comment by dionisiydk** [comment]
@guillep I agree that it should be part of refactoring engine but in better form. 
I introduced these methods in Calypso to be compatible with current version of Pharo (Pharo 6 at that time).

Let's continue this discussion in pharo issue.
